libnmstate/nm/active_connection.py

```diff
@@ -148,13 +144,27 @@ def wait(ctx, nm_ac, nm_dev):
 
     def _activate_profile_callback(self, nm_client, result, _user_data):
         nm_ac = None
+        retry = _user_data
```

Comment:
You may just rename the `_user_data` to `retry` in the function argument.

Reply:
Done, thank you!